### PR TITLE
Makes emagged borgs immune to robotics console self destruct

### PR DESCRIPTION
**Pull request by Alberyk**
What the title says.

### CODE REVIEW

**Comment by Lohikar** [comment]
Why is this bypassing dev?

**Comment by BurgerLUA** [comment]
Little iffy on the balance of this. Personally, I think there should be a cosmetic notification that tells you that the borg has been tampered with in some way.

**Comment by Alberyk** [comment]
For the same reason that the pr that made emagged borgs immune to locked bypassed dev.

**Comment by Arrow768** [comment]
I also think that there should be some message to the user and the borg.
By the time they blow up the borg, shit must have hit the fan -> No additional meta information is gained.

File: code/game/machinery/computer/robot.dm [hunk 1]
```diff
@@ -62,6 +62,11 @@
 		if(target.mind && target.mind.special_role && target.emagged)
 			target << "Extreme danger.  Termination codes detected.  Scrambling security codes and automatic AI unlink triggered."
 			target.ResetSecurityCodes()
+
+		if(target.emagged)
+			user << "Access Denied. Safeties protocols are disabled."
```
Comment (Lohikar) [inline code/game/machinery/computer/robot.dm]: Safety should be singular here.

**Pull request merged**